easybuild/easyconfigs/p/pymatgen/pymatgen-2023.12.18-foss-2023a.eb

```diff
@@ -12,8 +12,8 @@ toolchain = {'name': 'foss', 'version': '2023a'}
 builddependencies = [
     ('hatchling', '1.18.0'),
     ('hypothesis', '6.82.0'),  # required for tests
-    ('maturin', '1.1.0'),
     ('poetry', '1.5.1'),
+    ('maturin', '1.1.0'),
```

Comment:
you meant to remove this? you just moved it down 2 lines

Reply:
fixed in 2e4f299